handlers/workloads.go

```diff
@@ -178,7 +178,7 @@ func PodLogs(w http.ResponseWriter, r *http.Request) {
 	queryParams := r.URL.Query()
 
 	// Get business layer
-	business, err := getBusiness(r)
+	businessLayer, err := getBusiness(r)
```

Comment:
All handlers are using same naming pattern on this code:
![image](https://user-images.githubusercontent.com/1662329/96549612-04435e00-12b0-11eb-82d5-e2a7297c1d22.png)

I'd suggest to not renaming existing variables, otherwise code style will change.

Reply:
I did this because, in this context, business can mean both the variable and the package name (and I need types defined inside the package).

Can you think another solution that would make sense here?

Reply:
I haven't review your PR in deep, just that came to my attention in a first look.
If you have a clash it seems you are adding some model into the business package that you need to import.
You can use the model package for that, it's where typically all models used by Kiali on this logic are used.

Reply:
I feel like those are more internal types to the package, but sounds good, will move them there.

Reply:
On the contrary. The Kiali model exposed to the UI is mainly located under the package "model".
At least for all logic related App/Service/Workload, Istio Validations and Istio Config.

Reply:
The LogOptions is more about the type expected by the business method than any Model used for a return.  I don't think it should be moved to models.  In my opinion the "business" variable convention is the issue, it shadows the package name and seems like bad style.  I actually think the variable name should change or, in this instance we could preserve the variable name and alias the package like: `businesspkg "github.com/kiali/kiali/business"`.

Reply:
I like changing the package name, sounds like a good compromise.